src/popover/popover-default-animation.jsx

```diff
@@ -0,0 +1,140 @@
+import Transitions from '../styles/transitions';
+import React from 'react';
+import PropTypes from '../utils/prop-types';
+import StylePropable from '../mixins/style-propable';
+import DefaultRawTheme from '../styles/raw-themes/light-raw-theme';
+import ThemeManager from '../styles/theme-manager';
+import Paper from '../paper';
+
+const PopoverAnimation = React.createClass({
```

Comment:
The variable name doesn't match with the component name.
I think that it should be the same, that's simpler when debugging.